sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -891,6 +894,12 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
        spval,recname(17),coarsepm(ista_2l,jsta_2l,1,1),lm)
        call read_netcdf_3d_para(ncid2d,im,jm,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
```

Comment:
@jaymes-kenyon In the line 895, 897, 899, 901, change "ncid2d" into "ncid3d".

Reply:
Thanks!  The latest commit makes these changes.  I am just confirming that I modified line 895, which existed prior to this PR. This line affects the handling of the "ext550" netCDF array. @EricJames-NOAA

Reply:
@jaymes-kenyon I checked RRFS history files. The variable "ext550" is from  file *phy*. So the line 895 should be kept as "nid2dd". Sorry for misleading. @EricJames-NOAA may chime in.

Reply:
@WenMeng-NOAA Yes I think line 895 should stay as ncid2d.  I think just lines 897, 899, and 901 should be ncid3d.  @jaymes-kenyon did you have a chance to test the new code for RRFS to see if the new fields come out as expected?

Reply:
@EricJames-NOAA Thanks for confirming. I am working on a test with model files provided by @jaymes-kenyon and will have both of you to verify results soon.

Reply:
The mean/max/min values of the new fields look reasonable with wgrib2.  Thanks for the test @WenMeng-NOAA!

Reply:
@WenMeng-NOAA — I would suggest updates to precision settings in a possible separate / future PR.  This will give NCAR collaborators a chance to look at this new data and see if it meets their requirements.  Feel free to merge this PR when ready, then we can install the updates into the real-time RRFS runs.